src/cascadia/TerminalSettingsModel/Microsoft.Terminal.Settings.Model.rc

```diff
@@ -0,0 +1,31 @@
+// Microsoft Visual C++ generated resource script.
+//
+#include "resource.h"
+
+#define APSTUDIO_READONLY_SYMBOLS
+/////////////////////////////////////////////////////////////////////////////
+//
+// Generated from the TEXTINCLUDE 2 resource.
+//
+#include "winres.h"
+
+/////////////////////////////////////////////////////////////////////////////
+#undef APSTUDIO_READONLY_SYMBOLS
+
+/////////////////////////////////////////////////////////////////////////////
+// English (United States) resources
+
+#if !defined(AFX_RESOURCE_DLL) || defined(AFX_TARG_ENU)
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+#pragma code_page(1252)

```

Comment:
I believe we should use CP_UTF8 here (65001), just in case that any of our JSON files include non-ASCII characters. Raymond has a blog post about this: https://devblogs.microsoft.com/oldnewthing/20190607-00/?p=102569

tl;dr:
* Use `#pragma code_page(65001)`
* Don't use a BOM on the file (you can check with VS Code in the bottom-right)

I may be mistaken, but I believe you can strip the comments in this file, if you'd like. We aren't really using "APSTUDIO" anymore after all. 😅

Reply:
FWIW I think that VS uses those comments _still_. It re-adds them.

as for the codepage, I don’t believe it has any impact on RCDATA. RCDATA is explicitly used for untyped binary data. If the encoding changed data included directly by file, it would be a disaster

Reply:
However, it WILL make it better for the rest of the file! We should still consider it